dotnet/dynamodb/README.md

```diff
@@ -397,6 +397,37 @@ It takes the following options:
 The update sets the **Order_Status** field of the item.
 It does not check whether the *ID* applies to a customer, order, or product.
 
+## Modifying a table item using the DynamoDB DataModel
```

Comment:
DynamoDB -> Amazon DynamoDB for first mention in each code file.

Reply:
It's going to be folded into the larger readme, which already has the full name on first use.